src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/cli/HistoricMovesCommand.kt

```diff
@@ -4,13 +4,13 @@ import org.springframework.boot.autoconfigure.condition.ConditionalOnNotWebAppli
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.pecs.jpc.domain.price.Supplier
 import uk.gov.justice.digital.hmpps.pecs.jpc.service.HistoricMovesProcessingService
-import uk.gov.justice.digital.hmpps.pecs.jpc.util.ClosedRangeLocalDate
+import uk.gov.justice.digital.hmpps.pecs.jpc.util.DateRange
```

Comment:
This is purely a rename (refactor).  There are no functional changes to any of the files impacted by this PR.